policies/communication_channels.md

```diff
@@ -0,0 +1,63 @@
+# Communication channels in the Astropy Project
+
+## Preface
+
+Communication in the Astropy Project is governed by the [Community
+Code of Conduct](https://www.astropy.org/code_of_conduct.html), which
+takes precedence over anything written here. This goal of this policy is
+to write down the technical details of how email lists, Slack
+channels, forums etc. are set up.
+
+While we aim to be an inclusive and welcoming community, we
+recognize that not all users want to be involved in the technical
+details of building and running a library like Astropy. Therefore, some
+communication channels that are more geared towards users and others
+to developers. Also, in a project this size, certain issues that
+arise require privacy (e.g. personal or financial information is
+legally protected in many countries). Thus, there are communication
+channels geared towards or exclusively for developers, maintainers, or
+members of certain committees.
```

Comment:
```suggestion
members of certain committees/roles.
```
Clarifying that some of the groups like the release team isn't a "committee" but this statement still applies